servers/v3-proxy-api/src/routes/validations/FetchSchema.ts

```diff
@@ -0,0 +1,101 @@
+import { Schema } from 'express-validator';
+
+/**
+ * Note: this type is manually documented, since the
+ * Schema validator doesn't infer types.
+ */
+export type V3FetchParams = {
+  access_token: string;
+  consumer_key: string;
+  chunk?: number;
+  count: number;
+  offset: number;
+  shares?: boolean;
+  updatedBefore?: number;
+};
+
+/**
+ * Schema for valid V3 GET/POST requests.
+ * Depending on the method, checkSchema looks
+ * for the data in the query or the body.
+ *
+ * This gives us some safety from bad user input values
+ * and limits the cases we have to handle downstream.
+ */
+export const V3FetchSchema: Schema = {
+  access_token: {
+    isString: true,
+    notEmpty: {
+      errorMessage: '`access_token` cannot be empty',
+    },
+  },
+  consumer_key: {
+    isString: true,
+    notEmpty: {
+      errorMessage: '`consumer_key` cannot be empty',
+    },
+  },
+  offset: {
+    // offet of the items to return.
+    default: {
+      options: 0,
+    },
+    isInt: {
+      options: {
+        min: 0,
+      },
+      errorMessage: '`offset` cannot be negative',
+    },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
+  },
+  count: {
+    optional: true,
+    isInt: {
+      options: {
+        min: 1,
+        max: 5000,
+      },
+    },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
+  },
+  chunk: {
+    default: {
+      options: 0,
+    },
+    isInt: {
+      options: {
+        min: 0,
+        max: 5000,
+      },
+    },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
+  },
+  updatedBefore: {
```

Comment:
Need to implement this in a followup pr..

Reply:
https://mozilla-hub.atlassian.net/browse/POCKET-9721

Reply:
this should be the same as `since` I think, no? can just reuse